PL release notes for 791xx (#1399)
harness/developer-hub: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md

If you don't see a new feature or enhancement in your Harness account, it might be behind a Feature Flag. Contact [Harness Support](mailto:support@harness.io) to enable the feature.

### April 21, 2023, version 791xx

#### What's new

> - Switching accounts now loads accounts on scroll. This increases the modal load time if there are more accounts to be loaded. (PL-30467)

Reviewer: ```suggestion
- Switching accounts now loads accounts on scroll. This increases the modal load time when there are more accounts to load. (PL-30467)
```
